tests/params/test_outputs.py

```diff
@@ -86,11 +87,6 @@ def test_surface_output():
             output_fields=["Cp", "qcriterion"],
         )
 
-    with pytest.raises(pd.ValidationError):
-        output = SurfaceOutput(
-            output_fields=["invalid_field", "qcriterion"],
-        )
-
```

Comment:
why do we need to remove this test?

Reply:
The validation is deferred to a root validator so by just instantiating the `SurfaceOutput` obj it is no enough to tell "invalid_field" is invalid. e.g. a user defined field called "invalid_field".

Reply:
Yes, I added invalid field test.